keyboards/handwired/pterodactyl/info.json

```diff
@@ -0,0 +1,12 @@
+{
+  "keyboard_name": "Pterodactyl",
+  "url": "",
+  "maintainer": "qmk",
+  "width": 17,
+  "height": 8,
+  "layouts": {
+    "LAYOUT_pterodactyl": {
+      "layout": [{"label":"k00", "x":0, "y":0}, {"label":"k01", "x":1, "y":0}, {"label":"k02", "x":2, "y":0}, {"label":"k03", "x":3, "y":0}, {"label":"k04", "x":4, "y":0}, {"label":"k05", "x":5, "y":0}, {"label":"k10", "x":0, "y":1}, {"label":"k11", "x":1, "y":1}, {"label":"k12", "x":2, "y":1}, {"label":"k13", "x":3, "y":1}, {"label":"k14", "x":4, "y":1}, {"label":"k15", "x":5, "y":1}, {"label":"k20", "x":0, "y":2}, {"label":"k21", "x":1, "y":2}, {"label":"k22", "x":2, "y":2}, {"label":"k23", "x":3, "y":2}, {"label":"k24", "x":4, "y":2}, {"label":"k25", "x":5, "y":2}, {"label":"k30", "x":0, "y":3}, {"label":"k31", "x":1, "y":3}, {"label":"k32", "x":2, "y":3}, {"label":"k33", "x":3, "y":3}, {"label":"k34", "x":4, "y":3}, {"label":"k35", "x":5, "y":3}, {"label":"k40", "x":0, "y":4}, {"label":"k41", "x":1, "y":4}, {"label":"k42", "x":2, "y":4}, {"label":"k43", "x":3, "y":4}, {"label":"k44", "x":4, "y":4}, {"label":"k55", "x":6, "y":5}, {"label":"k50", "x":7, "y":5}, {"label":"k54", "x":7, "y":6}, {"label":"k53", "x":5, "y":6, "h":2}, {"label":"k52", "x":6, "y":6, "h":2}, {"label":"k51", "x":7, "y":7}, {"label":"k06", "x":11, "y":0}, {"label":"k07", "x":12, "y":0}, {"label":"k08", "x":13, "y":0}, {"label":"k09", "x":14, "y":0}, {"label":"k0A", "x":15, "y":0}, {"label":"k0B", "x":16, "y":0}, {"label":"k16", "x":11, "y":1}, {"label":"k17", "x":12, "y":1}, {"label":"k18", "x":13, "y":1}, {"label":"k19", "x":14, "y":1}, {"label":"k1A", "x":15, "y":1}, {"label":"k1B", "x":16, "y":1}, {"label":"k26", "x":11, "y":2}, {"label":"k27", "x":12, "y":2}, {"label":"k28", "x":13, "y":2}, {"label":"k29", "x":14, "y":2}, {"label":"k2A", "x":15, "y":2}, {"label":"k2B", "x":16, "y":2}, {"label":"k36", "x":11, "y":3}, {"label":"k37", "x":12, "y":3}, {"label":"k38", "x":13, "y":3}, {"label":"k39", "x":14, "y":3}, {"label":"k3A", "x":15, "y":3}, {"label":"k3B", "x":16, "y":3}, {"label":"k47", "x":12, "y":4}, {"label":"k48", "x":13, "y":4}, {"label":"k49", "x":14, "y":4}, {"label":"k4A", "x":15, "y":4}, {"label":"k4B", "x":16, "y":4}, {"label":"k5B", "x":9, "y":5}, {"label":"k56", "x":10, "y":5}, {"label":"k57", "x":9, "y":6}, {"label":"k5A", "x":9, "y":7}, {"label":"k59", "x":10, "y":6, "h":2}, {"label":"k58", "x":11, "y":6, "h":2}]
+    }
+  }
+}
```

Comment:
```suggestion
{
    "keyboard_name": "Pterodactyl",
    "url": "",
    "maintainer": "qmk",
    "width": 17,
    "height": 8,
    "layouts": {
        "LAYOUT_pterodactyl": {
            "layout": [
                {"x": 0, "y": 0},
                {"x": 1, "y": 0},
                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},

                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0},
                {"x": 14, "y": 0},
                {"x": 15, "y": 0},
                {"x" :16, "y": 0},

                {"x": 0, "y": 1},
                {"x": 1, "y": 1},
                {"x": 2, "y": 1},
                {"x": 3, "y": 1},
                {"x": 4, "y": 1},
                {"x": 5, "y": 1},

                {"x": 11, "y": 1},
                {"x": 12, "y": 1},
                {"x": 13, "y": 1},
                {"x": 14, "y": 1},
                {"x": 15, "y": 1},
                {"x": 16, "y": 1},

                {"x": 0, "y": 2},
                {"x": 1, "y": 2},
                {"x": 2, "y": 2},
                {"x": 3, "y": 2},
                {"x": 4, "y": 2},
                {"x": 5, "y": 2},

                {"x": 11, "y": 2},
                {"x": 12, "y": 2},
                {"x": 13, "y": 2},
                {"x": 14, "y": 2},
                {"x": 15, "y": 2},
                {"x": 16, "y": 2},

                {"x": 0, "y": 3},
                {"x": 1, "y": 3},
                {"x": 2, "y": 3},
                {"x": 3, "y": 3},
                {"x": 4, "y": 3},
                {"x": 5, "y": 3},

                {"x": 11, "y": 3},
                {"x": 12, "y": 3},
                {"x": 13, "y": 3},
                {"x": 14, "y": 3},
                {"x": 15, "y": 3},
                {"x": 16, "y": 3},

                {"x": 0, "y": 4},
                {"x": 1, "y": 4},
                {"x": 2, "y": 4},
                {"x": 3, "y": 4},
                {"x": 4, "y": 4},

                {"x": 5, "y": 6, "h": 2},

                {"x": 11, "y": 6, "h": 2},
                {"x": 12, "y": 4},
                {"x": 13, "y": 4},
                {"x": 14, "y": 4},
                {"x": 15, "y": 4},
                {"x": 16, "y": 4},

                {"x": 6, "y": 5},
                {"x": 7, "y": 5},

                {"x": 9, "y": 5},
                {"x": 10, "y": 5},

                {"x": 7, "y": 6},

                {"x": 9, "y": 6},

                {"x": 6, "y": 6, "h": 2},
                {"x": 7, "y": 7},

                {"x": 9, "y": 7},
                {"x": 10, "y": 6, "h": 2}
            ]
        }
    }
}
```
This is out of order with respect to the LAYOUT macro. I've fixed it and cleaned it up a bit.